portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +34,35 @@ function createCommentElement(userName, comment) {
   divElement.innerText = userName + ': ';
   divElement.appendChild(pElement);
   return divElement;
-}
\ No newline at end of file
+}
+
+function getLoggedInStatus() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    const loggedIn = json.loggedInStatus;
+
+    if (loggedIn){
+      const pElement = document.getElementById('display-user');
+      pElement.style.display = "block";
```

Comment:
Another option is to have 2 separate css classes one of on and one for off and toggle between them.

Reply:
Done: dbe9632